static/includes/AddAdminUserAccount.md

```diff
@@ -1,36 +1,37 @@
 &NewLine;
 
-All CORE systems migrating to SCALE, and all Angelfish and early Bluefin releases of SCALE upgrading to 22.12.3+ or to later SCALE major versions should create and begin using an admin user instead of the root user. 
-After migrating or upgrading from CORE or a pre-SCALE 22.12.3 release to a later SCALE release, use this procedure to create the Local Administrator user.
+All CORE systems migrating to SCALE, and all early SCALE releases such as Angelfish and pre-22.12.3 Bluefin, that migrate or upgrade to later SCALE major versions must create and begin using an admin user instead of the root user.
+After migrating or upgrading from CORE or a pre-SCALE 22.12.3 releases to a later SCALE release, use this procedure to create the administrator user.
 
 Go to **Credentials > Local Users** and click **Add**.
 
-Enter the name to use for the administrator account. For example, *admin*.
-You can create multiple admin users with any name and assign each different administration privileges.
+Enter the name to use for the administrator account. For example, **truenas_admin**, **admin**, etc.
```

Comment:
```suggestion
Enter the name to use for the administrator account. For example, **truenas_admin**, **tnas-admin**, etc.
As a security recommendation, try to create unique account names that aren't easy to guess.
```

Reply:
they cannot enter tnas_admin, they have to enter truenas_admin as that is the default user created... unless eng has changed this in the past week. the two possible system-created administrators are truenas_admin or admin based on how you get to 24.10.